vulns/CVE-2020-10749.json

```diff
@@ -1,52 +1,44 @@
 {
-    "id": "CVE-2020-10749",
-    "modified": "2020-05-27T18:18:01Z",
-    "published": "2020-05-27T18:18:01Z",
-    "summary": "Malicious container can reconfigure the host to redirect part or all of the IPv6 traffic of the host to the attacker-controlled container",
-    "details": "A cluster configured to use an affected container networking implementation is susceptible to man-in-the-middle (MitM) attacks. By sending “rogue” router advertisements, a malicious container can reconfigure the host to redirect part or all of the IPv6 traffic of the host to the attacker-controlled container. Even if there was no IPv6 traffic before, if the DNS returns A (IPv4) and AAAA (IPv6) records, many HTTP libraries will try to connect via IPv6 first then fallback to IPv4, giving an opportunity to the attacker to respond.",
-    "affected": [
+  "id": "CVE-2020-10749",
+  "modified": "2020-05-27T19:32:29Z",
+  "published": "2020-05-27T19:32:29Z",
+  "summary": "IPv4 only clusters susceptible to MitM attacks via IPv6 rogue router advertisements",
+  "details": "A vulnerability was found in all versions of containernetworking/plugins before version 0.8.6, that allows malicious containers in Kubernetes clusters to perform man-in-the-middle (MitM) attacks. A malicious container can exploit this flaw by sending rogue IPv6 router advertisements to the host or other containers, to redirect traffic to the malicious container.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "/containernetworking/plugins"
```

Comment:
Isn't it kubelet?

Reply:
yes, we fixed it manually on master

Reply:
So, do you mean `/containernetworking/plugins` is correct?

Reply:
I mean kubelet is correct

Reply:
OK, but this PR updates many fields. Do you mean to roll them back and leave "modified" only?

Reply:
yes

Reply:
I got it now. Sounds good👍

Reply:
modified date is updated now.